core/src/main/java/com/klaviyo/coresdk/KlaviyoConfig.kt

```diff
@@ -46,15 +46,15 @@ object KlaviyoConfig {
             this.applicationContext = context
         }
 
-        fun networkTimeout(networkTimeout: Int)  = apply {
+        fun networkTimeout(networkTimeout: Int) = apply {
             if (networkTimeout < 0) {
                 // TODO: When Timber is installed, log warning here
```

Comment:
Can we use Timber if the app might use it?

Reply:
Ya if we make Timber a debug dependency only it won't be included in release builds

Reply:
Pretty sure Timber has its own utilities to further ensure logs are only written on debug trees too

Reply:
Ok sounds good.